profilers/execution_profiler/profiler_worker.py

```diff
@@ -146,36 +198,35 @@ def main():
 
 
     #send output file back to the scheduler machine
+    global username,password,ssh_port,num_retries
+
     master_IP   = os.environ['HOME_NODE']
     username    = config['AUTH']['USERNAME']
     password    = config['AUTH']['PASSWORD']
     ssh_port    = int(config['PORT']['SSH_SVC'])
     num_retries = int(config['OTHER']['SSH_RETRY_NUM'])
-    retry       = 0
+
 
     local_profiler_path    = os.path.join(os.path.dirname(__file__), 'profiler_' + nodename + '.txt')
     remote_path = "/centralized_scheduler/profiler_files/"
 
-    # cmd = "sshpass -p " + password + " scp -P " + str(ssh_port) + " -o StrictHostKeyChecking=no %s %s:%s" % (local_profiler_path,master_IP, remote_path)
-    # print("cmd:", cmd)
-    # os.system(cmd)
-
     if path.isfile(local_profiler_path):
-        client = paramiko.SSHClient()
-        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        while retry < num_retries:
-            try:
-                client.connect(master_IP, username=username, password=password, port=ssh_port)
-                scp = SCPClient(client.get_transport())
-                scp.put(local_profiler_path, remote_path)
-                scp.close()
-                # os.remove(local_profiler_path)
-                print('execution profiling data transfer complete\n')
-                break
-            except:
-                print('SSH Connection refused, will retry in 2 seconds')
-                time.sleep(2)
-                retry += 1
+        transfer_data(master_IP,username,password,local_profiler_path, remote_path)
+        # client = paramiko.SSHClient()
```

Comment:
@caravansary83 if we are not using them. maybe its better to just delete them for making the code cleaner